--- src/Makefile.orig	2014-11-05 10:20:08.878745475 -0800
+++ src/Makefile	2014-11-05 10:21:03.849151900 -0800
@@ -32,14 +32,14 @@
 # modified (except for -m486 on non-Intel x86 platforms).
 
 CFLAGS		= -Wall #-O2 #-m486
-DIRS		= -DWORKDIR=\"$(WORKDIR)\" \
+DIRS		= -DWORKDIR=\"$(WORKDIR)\" -DLOCKDIR=\"$(LOCKDIR)\" \
 		  -DLOGDIR=\"$(LOGDIR)\" -DEXECDIR=\"$(TARGET)\"
 LDOPTS		= #-static
 
 # you may want to change this to point to your ncurses include directory
 # if the ncurses include files are not in the default location.
 
-INCLUDEDIR	= -I/usr/include/ncurses -I../support
+INCLUDEDIR	= $(CPPFLAGS) -I../support
 
 # You can uncomment this one to disable the backspace key in input fields.
 # This means you must use the Del key or Ctrl+H combination to erase the
@@ -116,7 +116,7 @@
 	make -C ../support
 
 %.o: %.c *.h version
-	$(CC) $(CFLAGS) $(DIRS) $(INCLUDEDIR) $(VERSION) $(PLATFORM) $(PROF) $(DEBUG) $(EXECPERM) $(BSSETTING) -c -o $*.o $<
+	$(CC) $(CFLAGS) $(DIRS) $(INCLUDEDIR) $(VERSION) $(PLATFORM) $(PROF) $(DEBUG) $(EXECPERM) $(BSSETTING) -c  $<
 
 rvnamed: rvnamed.o getpath.o
 	$(CC) $(LDOPTS) $(PROF) -o rvnamed rvnamed.o getpath.o
